src/Components/Publishing/VideoPlayer/Timestamp.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import styled from "styled-components"
+import { Fonts } from "../Fonts"
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  duration: number
+  currentTime: number
+}
+
+const formatTime = (time) => {
```

Comment:
We will need to format the time in the ArticleCard component too if we aren't saving duration as a string -- maybe a good idea to move this into constants.

Reply:
Ah yeah, the player doesn't actually use the duration from the article. It picks up the exact duration here once video metadata is loaded: https://github.com/artsy/reaction/pull/408/files#diff-fb979d99f63804b1dd7cc80ddab6a9a0R45

I'll likely keep ^ to keep the player generic but good call that for ArticleCard we'll want to make the Positron side be able to do the same for article saving. Good idea to move this to a util to anticipate that. 👍